projects/solar/src/data_tx.c

```diff
@@ -33,7 +33,7 @@ static void prv_data_tx(SoftTimerId timer_id, void *context) {
     data_store_get_is_set(data_point, &data_is_set);
     if (data_is_set) {
       data_store_get(data_point, &data_value);
-      if (CAN_TRANSMIT_SOLAR_DATA((uint32_t)data_point, data_value) ==
+      if (CAN_TRANSMIT_SOLAR_DATA_5_MPPTS((uint32_t)data_point, data_value) ==
```

Comment:
Here you need to swap between sending `SOLAR_DATA_5_MPPTS` and `SOLAR_DATA_6_MPPTS` depending on which board we're on. You might need to add a `SolarMpptCount` to `DataTxSettings` to get the info into the module.